rllib/core/rl_module/rl_module.py

```diff
@@ -302,6 +302,42 @@ def get_state(self) -> Mapping[str, Any]:
     def set_state(self, state_dict: Mapping[str, Any]) -> None:
         """Sets the state dict of the module."""
 
+    def serialize(self) -> Mapping[str, Any]:
```

Comment:
I think this method should actually have a default implementation in here that is consistent with its deserialize counter-part.

Reply:
done